src/box/errcode.h

```diff
@@ -115,286 +115,286 @@ struct errcode_record {
+	_(ER_KEY_PART_TYPE, 18,			"Supplied key field type does not match index part type", "fieldno", UINT, "part_type", STRING) \
+	_(ER_EXACT_MATCH, 19,			"Invalid key part count in an exact match", "expected", UINT, "actual", UINT) \
+	_(ER_INVALID_MSGPACK, 20,		"Invalid MsgPack", "issue", STRING) \
+	/* ER_PROC_RET, 21, Unused */ \
 	_(ER_TUPLE_NOT_ARRAY, 22,		"Tuple/Key must be MsgPack array") \
-	_(ER_FIELD_TYPE, 23,			"Tuple field %s type does not match one required by operation: expected %s, got %s") \
-	_(ER_INDEX_PART_TYPE_MISMATCH, 24,	"Field %s has type '%s' in one index, but type '%s' in another") \
-	_(ER_UPDATE_SPLICE, 25,			"SPLICE error on field %s: %s") \
-	_(ER_UPDATE_ARG_TYPE, 26,		"Argument type in operation '%c' on field %s does not match field type: expected %s") \
-	_(ER_FORMAT_MISMATCH_INDEX_PART, 27,	"Field %s has type '%s' in space format, but type '%s' in index definition") \
-	_(ER_UNKNOWN_UPDATE_OP, 28,		"Unknown UPDATE operation #%d: %s") \
-	_(ER_UPDATE_FIELD, 29,			"Field %s UPDATE error: %s") \
+	_(ER_FIELD_TYPE, 23,			"Tuple field type does not match one required by operation", "field", STRING, "expected", STRING, "actual", STRING) \
+	_(ER_INDEX_PART_TYPE_MISMATCH, 24,	"Field type mismatch in indexes", "field", STRING, "type_1", STRING, "type_2", STRING) \
+	_(ER_UPDATE_SPLICE, 25,			"SPLICE error", "field", STRING, "details", STRING) \
+	_(ER_UPDATE_ARG_TYPE, 26,		"Argument type does not match expected field type", "operation", CHAR, "field", STRING, "field_type", STRING) \
+	_(ER_FORMAT_MISMATCH_INDEX_PART, 27,	"Field type mismatch in space format and index definition", "field", STRING, "space_type", STRING, "index_type", STRING) \
+	_(ER_UNKNOWN_UPDATE_OP, 28,		"Unknown UPDATE operation", "operation_index", INT, "operation", STRING) \
+	_(ER_UPDATE_FIELD, 29,			"Fields UPDATE error", "field", STRING, "details", STRING) \
 	_(ER_FUNCTION_TX_ACTIVE, 30,		"Transaction is active at return from function") \
-	_(ER_KEY_PART_COUNT, 31,		"Invalid key part count (expected [0..%u], got %u)") \
-	_(ER_PROC_LUA, 32,			"%s") \
-	_(ER_NO_SUCH_PROC, 33,			"Procedure '%.*s' is not defined") \
-	_(ER_NO_SUCH_TRIGGER, 34,		"Trigger '%s' doesn't exist") \
-	_(ER_NO_SUCH_INDEX_ID, 35,		"No index #%u is defined in space '%s'") \
-	_(ER_NO_SUCH_SPACE, 36,			"Space '%s' does not exist") \
-	_(ER_NO_SUCH_FIELD_NO, 37,		"Field %d was not found in the tuple") \
-	_(ER_EXACT_FIELD_COUNT, 38,		"Tuple field count %u does not match space field count %u") \
-	_(ER_FIELD_MISSING, 39,			"Tuple field %s required by space format is missing") \
+	_(ER_KEY_PART_COUNT, 31,		"Invalid key part count", "maximum", UINT, "actual", UINT) \
+	_(ER_PROC_LUA, 32,			"Error in Lua code", "details", STRING) \
+	_(ER_NO_SUCH_PROC, 33,			"Procedure is not defined", "procedure", STRING) \
+	_(ER_NO_SUCH_TRIGGER, 34,		"Trigger doesn't exist", "trigger", STRING) \
+	_(ER_NO_SUCH_INDEX_ID, 35,		"Index id is not found", "index_id", UINT, "space", STRING) \
```

Comment:
"Index doesn't exist". Probably, we should merge `NO_SUCH_INDEX_ID` and `NO_SUCH_INDEX_NAME` in the future.

Let's use the same naming for all errors that check object existence.

Reply:
Done.

<details>
<summary>diff</summary>

```diff
diff --git a/src/box/errcode.h b/src/box/errcode.h
index 43b053a14..a14f08f6d 100644
--- a/src/box/errcode.h
+++ b/src/box/errcode.h
@@ -147,11 +147,11 @@ struct errcode_record {
 	_(ER_FUNCTION_TX_ACTIVE, 30,		"Transaction is active at return from function") \
 	_(ER_KEY_PART_COUNT, 31,		"Key part count exceeds max allowed by the key definition", "max", UINT, "actual", UINT) \
 	_(ER_PROC_LUA, 32,			"Error in Lua function", "details", STRING) \
-	_(ER_NO_SUCH_PROC, 33,			"Function is not defined", "function", STRING) \
+	_(ER_NO_SUCH_PROC, 33,			"Function doesn't exits", "function", STRING) \
 	_(ER_NO_SUCH_TRIGGER, 34,		"Trigger doesn't exist", "trigger", STRING) \
-	_(ER_NO_SUCH_INDEX_ID, 35,		"Index id is not found", "index_id", UINT, "space", STRING) \
-	_(ER_NO_SUCH_SPACE, 36,			"Space does not exist", "space", STRING) \
-	_(ER_NO_SUCH_FIELD_NO, 37,		"Field is not found in the tuple", "fieldno", INT) \
+	_(ER_NO_SUCH_INDEX_ID, 35,		"Index doesn't exits", "index_id", UINT, "space", STRING) \
+	_(ER_NO_SUCH_SPACE, 36,			"Space doesn't exist", "space", STRING) \
+	_(ER_NO_SUCH_FIELD_NO, 37,		"Field doesn't exist", "fieldno", INT) \
 	_(ER_EXACT_FIELD_COUNT, 38,		"Tuple and space field count mismatch", "tuple_field_count", UINT, "space_field_count", UINT) \
 	_(ER_FIELD_MISSING, 39,			"Tuple field is required by space format", "field", STRING) \
 	_(ER_WAL_IO, 40,			"Failed to write to disk") \
@@ -159,23 +159,23 @@ struct errcode_record {
 	_(ER_ACCESS_DENIED, 42,			"Access denied") \
 	_(ER_CREATE_USER, 43,			"Failed to create user", "user", STRING, "details", STRING) \
 	_(ER_DROP_USER, 44,			"Failed to drop user", "user", STRING, "details", STRING) \
-	_(ER_NO_SUCH_USER, 45,			"User not found", "user", STRING) \
+	_(ER_NO_SUCH_USER, 45,			"User doesn't exist", "user", STRING) \
 	_(ER_USER_EXISTS, 46,                   "User already exists", "user", STRING) \
 	_(ER_CREDS_MISMATCH, 47,		"User not found or supplied credentials are invalid") \
 	_(ER_UNKNOWN_REQUEST_TYPE, 48,		"Unknown request type", "request_type", UINT) \
 	_(ER_UNKNOWN_SCHEMA_OBJECT, 49,		"Unknown object type", "object_type", STRING) \
 	_(ER_CREATE_FUNCTION, 50,		"Failed to create function", "function", STRING, "details", STRING) \
-	_(ER_NO_SUCH_FUNCTION, 51,		"Function does not exist", "function", STRING) \
+	_(ER_NO_SUCH_FUNCTION, 51,		"Function doesn't exist", "function", STRING) \
 	_(ER_FUNCTION_EXISTS, 52,		"Function already exists", "function", STRING) \
 	_(ER_BEFORE_REPLACE_RET, 53,		"Invalid return value of space:before_replace trigger: expected tuple or nil") \
 	_(ER_MULTISTATEMENT_TRANSACTION, 54,	"Can not perform in a multi-statement transaction", "action", STRING) \
 	_(ER_TRIGGER_EXISTS, 55,		"Trigger already exists", "trigger", STRING) \
 	_(ER_USER_MAX, 56,			"A limit on the total number of users has been reached", "limit", UINT) \
-	_(ER_NO_SUCH_ENGINE, 57,		"Space engine does not exist", "engine", STRING) \
+	_(ER_NO_SUCH_ENGINE, 57,		"Space engine doesn't exist", "engine", STRING) \
 	_(ER_RELOAD_CFG, 58,			"Can't set option dynamically", "option", STRING) \
 	_(ER_CFG, 59,				"Incorrect value for option", "option", STRING, "details", STRING) \
 	_(ER_SAVEPOINT_EMPTY_TX, 60,		"Can not set a savepoint in an empty transaction") \
-	_(ER_NO_SUCH_SAVEPOINT, 61,		"Can not rollback to savepoint: the savepoint does not exist") \
+	_(ER_NO_SUCH_SAVEPOINT, 61,		"Savepoint doesn't exist") \
 	_(ER_UNKNOWN_REPLICA, 62,		"Replica is not registered with replica set", "replica", STRING, "replicaset", STRING) \
 	_(ER_REPLICASET_UUID_MISMATCH, 63,	"Replica set UUID mismatch", "expected", STRING, "actual", STRING) \
 	_(ER_INVALID_UUID, 64,			"Invalid UUID", "uuid", STRING) \
@@ -196,7 +196,7 @@ struct errcode_record {
 	_(ER_ACTIVE_TRANSACTION, 79,		"Operation is not permitted when there is an active transaction ") \
 	_(ER_CURSOR_NO_TRANSACTION, 80,		"The transaction the cursor belongs to has ended") \
 	_(ER_CROSS_ENGINE_TRANSACTION, 81,	"A multi-statement transaction can not use multiple storage engines") \
-	_(ER_NO_SUCH_ROLE, 82,			"Role is not found", "role", STRING) \
+	_(ER_NO_SUCH_ROLE, 82,			"Role doesn't exist", "role", STRING) \
 	_(ER_ROLE_EXISTS, 83,			"Role already exists", "role", STRING) \
 	_(ER_CREATE_ROLE, 84,			"Failed to create role", "role", STRING, "details", STRING) \
 	_(ER_INDEX_EXISTS, 85,			"Index already exists", "index", STRING) \
@@ -259,37 +259,37 @@ struct errcode_record {
 	_(ER_CREATE_SEQUENCE, 142,		"Failed to create sequence", "sequence", STRING, "details", STRING) \
 	_(ER_ALTER_SEQUENCE, 143,		"Can't modify sequence", "sequence", STRING, "details", STRING) \
 	_(ER_DROP_SEQUENCE, 144,		"Can't drop sequence", "sequence", STRING, "details", STRING) \
-	_(ER_NO_SUCH_SEQUENCE, 145,		"Sequence does not exist", "sequence", STRING) \
+	_(ER_NO_SUCH_SEQUENCE, 145,		"Sequence doesn't exist", "sequence", STRING) \
 	_(ER_SEQUENCE_EXISTS, 146,		"Sequence already exists", "sequence", STRING) \
 	_(ER_SEQUENCE_OVERFLOW, 147,		"Sequence overflow", "sequence", STRING) \
-	_(ER_NO_SUCH_INDEX_NAME, 148,		"No index in space", "index", STRING, "space", STRING) \
+	_(ER_NO_SUCH_INDEX_NAME, 148,		"Index doesn't", "index", STRING, "space", STRING) \
 	_(ER_SPACE_FIELD_IS_DUPLICATE, 149,	"Space field is duplicate", "field", STRING) \
 	_(ER_CANT_CREATE_COLLATION, 150,	"Failed to initialize collation", "details", STRING) \
 	_(ER_WRONG_COLLATION_OPTIONS, 151,	"Wrong collation options", "details", STRING) \
 	_(ER_NULLABLE_PRIMARY, 152,		"Primary index of space can not contain nullable parts", "space", STRING) \
-	_(ER_NO_SUCH_FIELD_NAME_IN_SPACE, 153,	"Field is not found in space format", "field", STRING, "space", STRING) \
+	_(ER_NO_SUCH_FIELD_NAME_IN_SPACE, 153,	"Field doesn't exist", "field", STRING, "space", STRING) \
 	_(ER_TRANSACTION_YIELD, 154,		"Transaction has been aborted by a fiber yield") \
-	_(ER_NO_SUCH_GROUP, 155,		"Replication group does not exist", "group", STRING) \
+	_(ER_NO_SUCH_GROUP, 155,		"Replication group doesn't exist", "group", STRING) \
 	/* ER_SQL_BIND_VALUE, 156, Unused */ \
 	_(ER_SQL_BIND_TYPE, 157,		"Bind value type is not supported", "bind_value_type", STRING, "parameter", STRING) \
 	_(ER_SQL_BIND_PARAMETER_MAX, 158,	"SQL bind parameter limit reached", "limit", INT) \
 	_(ER_SQL_EXECUTE, 159,			"Failed to execute SQL statement", "details", STRING) \
 	_(ER_UPDATE_DECIMAL_OVERFLOW, 160,	"Decimal overflow", "operation", STRING, "field", STRING) \
-	_(ER_SQL_BIND_NOT_FOUND, 161,		"Parameter is not found in the statement", "parameter", STRING) \
+	_(ER_SQL_BIND_NOT_FOUND, 161,		"Parameter doesn't exist", "parameter", STRING) \
 	_(ER_ACTION_MISMATCH, 162,		"Action mismatch", "field", STRING, "field_action", STRING, "index_part_action", STRING) \
 	_(ER_VIEW_MISSING_SQL, 163,		"Space declared as a view must have SQL statement") \
 	_(ER_FOREIGN_KEY_CONSTRAINT, 164,	"Can not commit transaction: deferred foreign keys violations are not resolved") \
-	_(ER_NO_SUCH_MODULE, 165,		"Module does not exist", "module", STRING) \
-	_(ER_NO_SUCH_COLLATION, 166,		"Collation does not exist", "collation", STRING) \
+	_(ER_NO_SUCH_MODULE, 165,		"Module doesn't exist", "module", STRING) \
+	_(ER_NO_SUCH_COLLATION, 166,		"Collation doesn't exist", "collation", STRING) \
 	_(ER_CREATE_FK_CONSTRAINT, 167,		"Failed to create foreign key constraint", "constraint", STRING, "details", STRING) \
 	/* ER_DROP_FK_CONSTRAINT, 168, Unused */ \
-	_(ER_NO_SUCH_CONSTRAINT, 169,		"Constraint does not exist", "constraint", STRING, "space", STRING) \
+	_(ER_NO_SUCH_CONSTRAINT, 169,		"Constraint doesn't exist", "constraint", STRING, "space", STRING) \
 	_(ER_CONSTRAINT_EXISTS, 170,		"Constraint already exists", "constraint_type", STRING, "constraint", STRING, "space", STRING) \
 	_(ER_SQL_TYPE_MISMATCH, 171,		"Incompatible types", "actual", STRING, "expected", STRING) \
 	_(ER_ROWID_OVERFLOW, 172,		"Rowid is overflowed: too many entries in ephemeral space") \
 	_(ER_DROP_COLLATION, 173,		"Can't drop collation", "collation", STRING, "details", STRING) \
 	_(ER_ILLEGAL_COLLATION_MIX, 174,	"Illegal mix of collations") \
-	_(ER_SQL_NO_SUCH_PRAGMA, 175,		"Pragma does not exist", "pragma", STRING) \
+	_(ER_SQL_NO_SUCH_PRAGMA, 175,		"Pragma doesn't exist", "pragma", STRING) \
 	_(ER_SQL_CANT_RESOLVE_FIELD, 176,	"Can't resolve field", "field", STRING) \
 	_(ER_INDEX_EXISTS_IN_SPACE, 177,	"Index already exists", "index", STRING, "space", STRING) \
 	_(ER_INCONSISTENT_TYPES, 178,		"Inconsistent types", "expected", STRING, "actual", STRING) \
@@ -315,7 +315,7 @@ struct errcode_record {
 	_(ER_FUNC_INDEX_FUNC, 198,		"Failed to build a key for functional index", "index", STRING, "space", STRING, "details", STRING) \
 	_(ER_FUNC_INDEX_FORMAT, 199,		"Key format doesn't match one defined in functional index", "index", STRING, "space", STRING, "details", STRING) \
 	_(ER_FUNC_INDEX_PARTS, 200,		"Wrong functional index definition", "details", STRING) \
-	_(ER_NO_SUCH_FIELD_NAME, 201,		"Field was not found in the tuple", "field", STRING) \
+	_(ER_NO_SUCH_FIELD_NAME, 201,		"Field doesn't exist", "field", STRING) \
 	_(ER_FUNC_WRONG_ARG_COUNT, 202,		"Wrong number of arguments", "function", STRING, "actual", INT, "min", INT, "max", INT) \
 	_(ER_BOOTSTRAP_READONLY, 203,		"Trying to bootstrap a local read-only instance as master") \
 	_(ER_SQL_FUNC_WRONG_RET_COUNT, 204,	"SQL expects exactly one return value", "function", STRING, "actual", INT)\
@@ -368,13 +368,13 @@ struct errcode_record {
 	_(ER_INVALID_AUTH_REQUEST, 251,		"Invalid auth request", "auth_method", STRING, "details", STRING) \
 	_(ER_WEAK_PASSWORD, 252,		"Password doesn't meet security requirements", "details", STRING) \
 	_(ER_OLD_PASSWORD, 253,			"Password must differ from last passwords", "history_length", INT) \
-	_(ER_NO_SUCH_SESSION, 254,		"Session does not exist", "sid", ULLONG) \
+	_(ER_NO_SUCH_SESSION, 254,		"Session doesn't exist", "sid", ULLONG) \
 	_(ER_WRONG_SESSION_TYPE, 255,		"Session type is not supported", "session_type", STRING) \
 	_(ER_PASSWORD_EXPIRED, 256,		"Password expired") \
 	_(ER_AUTH_DELAY, 257,			"Too many authentication attempts") \
 	_(ER_AUTH_REQUIRED, 258,		"Authentication required") \
 	_(ER_SQL_SEQ_SCAN, 259,			"Scanning is not allowed", "object", STRING) \
-	_(ER_NO_SUCH_EVENT, 260,		"Unknown event", "even", STRING) \
+	_(ER_NO_SUCH_EVENT, 260,		"Event doesn't exist", "even", STRING) \
 	_(ER_BOOTSTRAP_NOT_UNANIMOUS, 261,	"Replica chose a different bootstrap leader", "replica", STRING, "bootstrap_leader", STRING) \
 	_(ER_CANT_CHECK_BOOTSTRAP_LEADER, 262,	"Can't check who replica chose its bootstrap leader", "replica", STRING) \
 	_(ER_BOOTSTRAP_CONNECTION_NOT_TO_ALL, 263, "Some replica set members were not specified in box.cfg.replication") \
```

</details>

Reply:
Fixed other places that use `doesn't`.

<details>
<summary>diff</summary>

```diff
diff --git a/src/box/errcode.h b/src/box/errcode.h
index f3385d0e6..09a810a13 100644
--- a/src/box/errcode.h
+++ b/src/box/errcode.h
@@ -155,7 +155,7 @@ struct errcode_record {
 	_(ER_EXACT_FIELD_COUNT, 38,		"Tuple and space field count mismatch", "tuple_field_count", UINT, "space_field_count", UINT) \
 	_(ER_FIELD_MISSING, 39,			"Tuple field is required by space format", "field", STRING) \
 	_(ER_WAL_IO, 40,			"Failed to write to disk") \
-	_(ER_MORE_THAN_ONE_TUPLE, 41,		"Get() doesn't support partial keys and non-unique indexes") \
+	_(ER_MORE_THAN_ONE_TUPLE, 41,		"Get() does not support partial keys and non-unique indexes") \
 	_(ER_ACCESS_DENIED, 42,			"Access denied") \
 	_(ER_CREATE_USER, 43,			"Failed to create user", "user", STRING, "details", STRING) \
 	_(ER_DROP_USER, 44,			"Failed to drop user", "user", STRING, "details", STRING) \
@@ -313,7 +313,7 @@ struct errcode_record {
 	/* ER_CK_CONSTRAINT_FAILED, 196, Unused */ \
 	_(ER_SQL_COLUMN_COUNT, 197,		"Number of entries mismatch in left and right side of row expression", "left", UINT, "right", UINT) \
 	_(ER_FUNC_INDEX_FUNC, 198,		"Failed to build a key for functional index", "index", STRING, "space", STRING, "details", STRING) \
-	_(ER_FUNC_INDEX_FORMAT, 199,		"Key format doesn't match one defined in functional index", "index", STRING, "space", STRING, "details", STRING) \
+	_(ER_FUNC_INDEX_FORMAT, 199,		"Key format does not match one defined in functional index", "index", STRING, "space", STRING, "details", STRING) \
 	_(ER_FUNC_INDEX_PARTS, 200,		"Wrong functional index definition", "details", STRING) \
 	_(ER_NO_SUCH_FIELD_NAME, 201,		"Field does not exist", "field", STRING) \
 	_(ER_FUNC_WRONG_ARG_COUNT, 202,		"Wrong number of arguments", "function", STRING, "actual", INT, "min", INT, "max", INT) \
@@ -341,7 +341,7 @@ struct errcode_record {
 	_(ER_ELECTION_DISABLED, 224,		"Elections were turned off")\
 	_(ER_TXN_ROLLBACK, 225,			"Transaction was rolled back") \
 	_(ER_NOT_LEADER, 226,			"The instance is not a leader", "leader_id", UINT)\
-	_(ER_SYNC_QUEUE_UNCLAIMED, 227,		"The synchronous transaction queue doesn't belong to any instance")\
+	_(ER_SYNC_QUEUE_UNCLAIMED, 227,		"The synchronous transaction queue does not belong to any instance")\
 	_(ER_SYNC_QUEUE_FOREIGN, 228,		"The synchronous transaction queue belongs to other instance", "instance_id", UINT)\
 	_(ER_UNABLE_TO_PROCESS_IN_STREAM, 229,	"Unable to process request in stream", "request", STRING) \
 	_(ER_UNABLE_TO_PROCESS_OUT_OF_STREAM, 230, "Unable to process request out of stream", "request", STRING) \
@@ -366,7 +366,7 @@ struct errcode_record {
 	_(ER_UNKNOWN_AUTH_METHOD, 249,		"Unknown authentication method", "auth_method", STRING) \
 	_(ER_INVALID_AUTH_DATA, 250,		"Invalid auth data", "auth_method", STRING, "details", STRING) \
 	_(ER_INVALID_AUTH_REQUEST, 251,		"Invalid auth request", "auth_method", STRING, "details", STRING) \
-	_(ER_WEAK_PASSWORD, 252,		"Password doesn't meet security requirements", "details", STRING) \
+	_(ER_WEAK_PASSWORD, 252,		"Password does not meet security requirements", "details", STRING) \
 	_(ER_OLD_PASSWORD, 253,			"Password must differ from last passwords", "history_length", INT) \
 	_(ER_NO_SUCH_SESSION, 254,		"Session does not exist", "sid", ULLONG) \
 	_(ER_WRONG_SESSION_TYPE, 255,		"Session type is not supported", "session_type", STRING) \

```

</details>